src/developer/web-api/new-tracker.md

```diff
@@ -337,6 +337,45 @@ Tracker notes do not have their dedicated endpoint; they are exchanged as part o
 > One between `uid` or `username` field must be provided. If both are provided, only username is considered.
 
 
+### Program stage working lists { #webapi_working_list_filters } 
+
+The program stage working lists are used in the Capture app to present predetermined working lists that are relevant to a specific program stage in the tracker user interface. To interact with them, you'll need to use the */api/trackedEntityInstanceFilters* resource. These lists can be shared and follow the same sharing pattern as any other metadata object. When using the */api/sharing* the type parameter will be *programStageWorkingLists*.
+
+    /api/40/programStageWorkingLists
+
+
+##### Payload on CRUD operations to program stage working lists
+
+If you want to fetch or make changes to a program stage working list, besides the endpoint mentioned above, you'll need to provide a payload in the following format: 
+
+
+
+Table: Payload
+
+| Payload values | Description | Example |
+|---|---|---|
+| name | Name of the working list. Required. ||
+| description | A description of the working list. ||
+| program | Object containing the id of the program. Required. | { "id" : "uy2gU8kTjF"} |
+| programStage | Object containing the id of the program stage. Required. | { "id" : "oRySG82BKE6"} |
```

Comment:
```suggestion
| programStage | Object containing the id of the program stage. Required. | {"id" : "oRySG82BKE6"} |
```

I think most samples do not have whitespace between `{`. We should stick to one style 😄